client/src/app/core/repositories/base-repository.ts

```diff
@@ -291,7 +285,7 @@ export abstract class BaseRepository<V extends BaseViewModel, M extends BaseMode
         if (!this.viewModelSubjects[id]) {
             this.viewModelSubjects[id] = new BehaviorSubject<V>(this.viewModelStore[id]);
         }
-        return this.viewModelSubjects[id].pipe(filter(value => !!value));
+        return this.viewModelSubjects[id].asObservable();
```

Comment:
Why did you change this? Nearly in every repo or component we catch, if a value is existing. Therefore, it can be filtered through the `base-repository`, I think. Or do you need anywhere the value `undefined`?